server/src/filters/global-exception.filter.ts

```diff
@@ -0,0 +1,18 @@
+import { ArgumentsHost, Catch, Injectable } from '@nestjs/common';
+import { GqlExceptionFilter } from '@nestjs/graphql';
+
+import { ExceptionCapturerService } from 'src/integrations/exception-capturer/exception-capturer.service';
+
+@Catch()
+@Injectable()
+export class GlobalExceptionFilter implements GqlExceptionFilter {
+  constructor(
+    private readonly exceptionCapturerService: ExceptionCapturerService,
+  ) {}
+
+  catch(exception: unknown, host: ArgumentsHost) {
```

Comment:
we don't use host?

Reply:
We don't really need it actually, I'll remove it